Module 2 Exercises.ipynb

```diff
@@ -114,16 +297,224 @@
+              "      <td>NR</td>\n",
+              "      <td>T</td>\n",
+              "      <td>Y_LT15</td>\n",
+              "      <td>UKN12</td>\n",
+              "      <td>2023</td>\n",
+              "      <td>NaN</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>1061816</th>\n",
+              "      <td>NR</td>\n",
+              "      <td>T</td>\n",
+              "      <td>Y_LT15</td>\n",
+              "      <td>UKN13</td>\n",
+              "      <td>2023</td>\n",
+              "      <td>NaN</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>1061817</th>\n",
+              "      <td>NR</td>\n",
+              "      <td>T</td>\n",
+              "      <td>Y_LT15</td>\n",
+              "      <td>UKN14</td>\n",
+              "      <td>2023</td>\n",
+              "      <td>NaN</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>1061818</th>\n",
+              "      <td>NR</td>\n",
+              "      <td>T</td>\n",
+              "      <td>Y_LT15</td>\n",
+              "      <td>UKN15</td>\n",
+              "      <td>2023</td>\n",
+              "      <td>NaN</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>1061819</th>\n",
+              "      <td>NR</td>\n",
+              "      <td>T</td>\n",
+              "      <td>Y_LT15</td>\n",
+              "      <td>UKN16</td>\n",
+              "      <td>2023</td>\n",
+              "      <td>NaN</td>\n",
+              "    </tr>\n",
+              "  </tbody>\n",
+              "</table>\n",
+              "<p>1061820 rows × 6 columns</p>\n",
+              "</div>"
+            ],
+            "text/plain": [
+              "        unit sex     age    geo  year  pop\n",
+              "0         NR   F   TOTAL     AL  1990  NaN\n",
+              "1         NR   F   TOTAL    AL0  1990  NaN\n",
+              "2         NR   F   TOTAL   AL01  1990  NaN\n",
+              "3         NR   F   TOTAL  AL011  1990  NaN\n",
+              "4         NR   F   TOTAL  AL012  1990  NaN\n",
+              "...      ...  ..     ...    ...   ...  ...\n",
+              "1061815   NR   T  Y_LT15  UKN12  2023  NaN\n",
+              "1061816   NR   T  Y_LT15  UKN13  2023  NaN\n",
+              "1061817   NR   T  Y_LT15  UKN14  2023  NaN\n",
+              "1061818   NR   T  Y_LT15  UKN15  2023  NaN\n",
+              "1061819   NR   T  Y_LT15  UKN16  2023  NaN\n",
+              "\n",
+              "[1061820 rows x 6 columns]"
+            ]
+          },
+          "execution_count": 139,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "\n",
+        "inc_data[['freq', 'unit', 'direct', 'na_item', 'geo']] = inc_data['freq,unit,direct,na_item,geo\\TIME_PERIOD'].str.split(',', expand=True)\n",
```

Comment:
A lot of code for reading in and cleaning the different TSVs can be shared across the three datasets. You might consider reducing redundant code by creating a function that can be applied to all three.